mlox_base.txt

```diff
@@ -553,178 +553,6 @@ Bloodmoon.esm
 GUARDS.esp
 GUARD-Remover.esp
 
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;; @Advanced Herbalism [andoreth]

```

Comment:
We need to keep this, Advanced Herbalism has features the modern Graphic Herbalism doesn't have (seeds, planting them and herbalism skill)
Other herbalism mods can be removed with the note on MWSE/OpenMW you added.